### PR TITLE
Feat/journal

### LINKED ISSUE #36: feat/journal
## Value statement - a requirement, not a solution
As a user
I need to write journal entries
so that I can keep track of my behavior over time.

## Description

Journal_List
Journal_List_2
Journal_Entry

## Acceptance criteria
- [x] The user can navigate to the journal page from the homepage and results page
- [x] All previous entries can be seen on the list page.
- [x] If there is no entry, there is a message that asks user to enter a new note
- [x] New journal entries can be added
- [x] Previous entries can be deleted

## Tasks
- [x] write tests
- [x] create journal list component
- [x] create journal form component
- [x]  read from and save to local storage
- [x] add components to storybook
- [x] add components to app

## Size
large

### PR DESCRIPTION
**Pull request by karstenlenz**
Please review my code.

closes #36

### CODE REVIEW

File: src/Journal/JournalForm.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  const [title, setTitle] = useState('')
+  const [description, setDescription] = useState('')
+  const history = useHistory()
+
+  return (
+    <FormStyled onSubmit={handleSubmit}>
+      <HeadlineUnderline>
+        <h1>Eintrag erstellen</h1>
+      </HeadlineUnderline>
+      <DateHeadline>Datum</DateHeadline>
```
Comment (jennydely) [inline src/Journal/JournalForm.js:19]: I think Underline will be enough here, Normally there is just 1 underline I think, so headline is not needed to mention here

File: src/common/ModalConfirm.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+`
+const CloseButton = styled.button`
+  width: 40px;
+  height: 40px;
+  padding: 0;
+  position: absolute;
+  right: 0px;
+  top: 0px;
+  background: none;
+  border: none;
```
Comment (jennydely) [inline src/common/ModalConfirm.js:102]: Maybe you can use some global styles or common Dom Elements, so you don't have to double stylings.

**Review by jennydely — APPROVED** [review]
Looks great, just made some small comments

File: src/Journal/JournalForm.js (hunk 1 above)
Comment (karstenlenz) [reply src/Journal/JournalForm.js:19]: I also have underlines for Text-Links, so for me it made sense to differentiate. I'll think about the naming again, though. I might be able to make the nameing more consistent and easier to understand.

File: src/common/ModalConfirm.js (hunk 2 above)
Comment (karstenlenz) [reply src/common/ModalConfirm.js:102]: Good point! I merged them into one "IconButton" styled component.

**Pull request merged**